backend/vendor/assets/stylesheets/jquery.jstree/themes/spree/style.scss

```diff
@@ -0,0 +1,216 @@
+/*
+ * jsTree Spree theme 1.0
+ */
+
+.jstree-spree {
+
+  li, li ul {
+    border-color: #ededed;
+    border-style: solid;
+  }
+
+  li {
+    margin-left: 26px;
+    border-width: 0 0 1px 0;
+
+    &:last-of-type {
+      border-bottom-width: 0;
+    }
+
+    ul {
+      border-width: 1px 0 0 0;
+    }
+  }
+
+  ins {
+    position: relative;
+    cursor: pointer;
+    width: 24px;
+    line-height: 1;
+    height: 38px;
+    vertical-align: middle;
+    font-family: 'Glyphicons Halflings';
+    color: #555;
```

Comment:
Color literals like `#555` should only be used in variable declarations; they should be referred to via variable everywhere else.